com.nonpolynomial.intiface_central.yaml

```diff
@@ -0,0 +1,70 @@
+# Credits to github/nickavem for the script, Doomsdayrs for edits
+
+app-id: com.nonpolynomial.intiface_central
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+
+command: run_intiface_central
+
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --system-talk-name=org.bluez
+
+modules:
+  - name: intiface
+    buildsystem: simple
+    build-commands:
+      # Launcher
+      - install -Dt /app/bin/ run_intiface_central
+
+      # App
+      - install -Dt /app/lib/intiface_central/ intiface_central
+      - cp -r data /app/lib/intiface_central/
+      - cp -r lib /app/lib/intiface_central/
+
+      # Metainfo
+      - install -Dt /app/share/metainfo/ com.nonpolynomial.intiface_central.metainfo.xml
+
+      # Desktop
+      - install -Dt /app/share/applications/ com.nonpolynomial.intiface_central.desktop
+
+      # Icon
+      - install -Dt /app/share/icons/hicolor/scalable/apps/ com.nonpolynomial.intiface_central.svg
+    sources:
+      # App
+      - type: archive
+        url: https://github.com/intiface/intiface-central/releases/download/v2.6.5/intiface-central-v2.6.5-linux-ubuntu-22.04-x64.zip
+        sha256: 8a15a03d003fe8f69c10f8e3702a818c97cb393196aa67852be2da3900219e49
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/intiface/intiface-central/releases/latest
+          version-query: .name | sub("^.*v"; "")
+          url-query: .assets[] | select(.name=="intiface-central-v" + $version + "-linux-ubuntu-22.04-x64.zip")
+            | .browser_download_url
+
+      # Icon
+      - type: file
+        url: https://raw.githubusercontent.com/intiface/intiface-central/bbe3e97f1509b657b187276bc403571a14e0d92f/assets/icons/intiface_central_icon.svg
+        dest-filename: com.nonpolynomial.intiface_central.svg
+        sha256: 1323486660dd2b235c510d264b112517ba93fd6cfaaac0dbc35279372f2900c4
+
+      # Metainfo
+      - type: file
+        url: https://raw.githubusercontent.com/intiface/intiface-central/1ec1c739b13054355f13b555b666c73dfbabe01c/linux/com.nonpolynomial.intiface_central.metainfo.xml
```

Comment:
screenshot links are 404 which is why you never should use a branch in the link, use a commit.

Reply:
https://github.com/intiface/intiface-central/pull/188